maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,735 @@
+            model_info["provider"] = "openai"
+        elif "anthropic" in str(type(model_obj)).lower():
+            model_info["provider"] = "anthropic"
+        elif hasattr(model_obj, 'provider_name'):
+            model_info["provider"] = model_obj.provider_name or "unknown"
+    except Exception as e:
+        scribe().debug(f"[MaximSDK] Error extracting model info: {e}")
+    
+    return model_info
+
+
+def convert_messages_to_maxim_format(pydantic_messages) -> list:
```

Ruff (0.12.2)

Missing type annotation for function argument `pydantic_messages`

(ANN001)